### PR TITLE
dev-java/cdegroot-db: min java 1.8

### PR DESCRIPTION
**Pull request by vaukai**
@fordfrog 
Maybe this one is good for roll-over stable keywords and removal of old ...

### CODE REVIEW

File: dev-java/cdegroot-db/cdegroot-db-0.08-r3.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+HOMEPAGE="http://cdegroot.com/software/db/"
+SRC_URI="http://cdegroot.com/software/db/download/com.${P/-/.}.tar.gz"
+
+LICENSE="MIT"
+SLOT="1"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="doc source"
+
+DEPEND=">=virtual/jdk-1.8:*"
+RDEPEND=">=virtual/jre-1.8:*"
```
Comment (monsieurp) [inline dev-java/cdegroot-db/cdegroot-db-0.08-r3.ebuild:20]:
```suggestion
DEPEND=">=virtual/jdk-1.8"
RDEPEND=">=virtual/jre-1.8"
```

**Review by monsieurp — CHANGES_REQUESTED** [review]

File: dev-java/cdegroot-db/cdegroot-db-0.08-r3.ebuild (hunk 1 above)
Comment (fordfrog) [reply dev-java/cdegroot-db/cdegroot-db-0.08-r3.ebuild:20]: @monsieurp without `:*` there's a warning (or at least it was) that the slot is not specified though it should, so we stick with this.

Comment (vaukai) [reply dev-java/cdegroot-db/cdegroot-db-0.08-r3.ebuild:20]: @fordfrog 
https://bugs.gentoo.org/785532

**Comment by fordfrog** [comment]
@vaukai thanks, will last-rite it, putting it on my list. closing this one as obsolete.

**Pull request closed without merging**